datarepo-clienttests/src/main/java/runner/TestRunner.java

```diff
@@ -537,6 +537,11 @@ public static boolean runTest(String configFileName, String outputParentDirName)
     boolean isFailure = false;
     for (int ctr = 0; ctr < testSuite.testConfigurations.size(); ctr++) {
       TestConfiguration testConfiguration = testSuite.testConfigurations.get(ctr);
+
+      // Lock namespace - if failure, then whole test run fails
+      if (testConfiguration.server.lockNamespace) {
+        KubernetesClientUtils.lockNamespace(testConfiguration.server);
+      }
```

Comment:
I think the namespace locking should be done by the deployment script instead of the main test runner. The reason is that I think locking is really a step in a particular type of deployment, rather than a generic thing for any test run.

I also think that the lockNamespace flag should be a parameter passed to the deployment script, instead of a property of the server directly.